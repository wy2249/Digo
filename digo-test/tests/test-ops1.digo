func digo_main() void {
  println("%d %d %d %d", 1 + 2, 1 - 2, 1 * 2, 100 /2)
  println("%d", 99)
  println("%d", 1 == 2)
  println("%d", 1 == 1)
  println("%d", 99)
  println("%d %d", 1 != 2, 1 != 1)
  println("%d %d %d %d", 1 < 2, 2 < 1, 1 <= 2, 1 <= 1, 2 <= 1)
}
