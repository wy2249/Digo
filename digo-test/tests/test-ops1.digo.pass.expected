3 -1 2 50
99
0
1
99
1 0
1 0 1 1
